docs/getting-started/requirements.md

```diff
@@ -71,14 +71,20 @@ To use each one you need to pass the proper flag to the execution. Prowler fro A
 
 #### Azure Active Directory scope
 
-Azure Active Directory (AAD) permissions required by the tool are the following:
+Microsoft Entra ID (AAD earlier) permissions required by the tool are the following:
 
 - `Directory.Read.All`
 - `Policy.Read.All`
 
 The best way to assign it is through the azure web console:
```

Comment:
```suggestion
The best way to assign it is through the azure web console:

```

Reply:
You have to include a new line for the numbered list to be formatted.

Reply:
Done